src/opendr/perception/object_detection_2d/detr/algorithm/util/misc.py

```diff
@@ -255,7 +255,7 @@ def log_every(self, iterable, print_freq, header=None, verbose=True, silent=Fals
                         time=str(iter_time), data=str(data_time)))
             i += 1
             end = time.time()
-        if not silent:
+        if verbose:
```

Comment:
the `silent` argument still serves a purpose? Doesn't appear to be used in the method

Reply:
Yes you are correct! Thank you for pointing this out, I removed the argument from the function.